examples/waspleau/ext/.waspignore

```diff
@@ -0,0 +1,3 @@
+# Ignore editor tmp files
```

Comment:
we could also potentially scaffold for user a .waspignore file that covers the most common editors. Do you think it makes sense? If yes, we could make an issue and mark it as a "good-beginner-issue", we are always in lack of these :D.

Reply:
I think that could be a good beginner issue. Would having editor ignores go into both the `.gitignore` and `.waspignore` perhaps?

Reply:
FYI @matijaSos , this .waspignore file you are looking at here is automatically generated when you create a new project, with these lines for ignoring editor tmp files.